pages/common/git-worktree.md

```diff
@@ -0,0 +1,15 @@
+# git worktree
+
+> Manage multiple working trees attached to the same repository.
+
+- Create a new folder with the specified branch checked out into it:
+
+`git worktree add {{path/to/folder}} {{branch}}`
```

Comment:
> Is it better to have them grouped by type of command, or listed in order of most commonly used to least commonly used?

The options should be ordered by the workflow. Usually we cite the most simple ones first, then slowly add complexity, building up on previous examples. In this case, we should put `add` first, then `list` and `prune` at last.

> What are your thoughts on:

I like the 2nd one.